frontend/src/screens/AccountsScreen.js

```diff
@@ -7,7 +7,7 @@ import fetchAccountsAction from '../actions/fetchAccountsAction';
 
 function AccountsScreen({ accounts, fetchAccounts, user }) {
   React.useEffect(
-    () => { fetchAccounts(user.token); }, [],
+    () => { fetchAccounts(user.token); }, [accounts],
```

Comment:
```suggestion
    () => { fetchAccounts(user.token); }, [],
```
This is causing infinite requests